features/networking/service.feature

```diff
@@ -154,3 +154,44 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-11645
+  @admin
+  Scenario: Create loadbalancer service
+    Given I have a project
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/e3e88a1b3706a7ede5c6cb7805fa72d3218a9998/networking/ping_for_pod_containerPort.json |
```

Comment:
@huiran0826 I assume you want to create loadbalancer svc, not pod here

Reply:
@weliang1 , Thanks for reviewing.  It created pod here firstly, then create loadbalancer svc  by next step "When I run the :create_service_loadbalancer client command with:"

Reply:
Is it purposely to use test file from a specific commit e3e88a1b3706a7ede5c6cb7805fa72d3218a9998?

Reply:
Thanks , fix it.